mix.exs

```diff
@@ -57,7 +57,7 @@ defmodule Miss.MixProject do
     [
       main: "readme",
       extras: ~w(README.md CHANGELOG.md),
-      logo: "misc/miss-elixir-logo.jpg",
+      logo: "priv/miss-elixir-logo.png",
```

Comment:
Why did you remove the `misc` directory?

Reply:
> Why did you remove the `misc` directory?

Assumed the standard `priv` directory common used for to store module asset. Let me know, I can revert back to original `misc` folder.

Reply:
Please move to `assets` and keep the original image.

Reply:
> Please move to `assets` and keep the original image.

Done. Please verify.